src/Ethereum/ContractCall.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "ContractCall.h"
+#include "ABI.h"
+#include "HexCoding.h"
+#include "uint256.h"
+#include <boost/algorithm/string/predicate.hpp>
+#include <iostream>
+
+using namespace TW;
+using namespace TW::Ethereum::ABI;
+using namespace std;
+
+using json = nlohmann::json;
+
+static void fillArray(Function& func, const string& type) {
```

Comment:
Private function in global namespace; better to use namespace TW::Ethereum::ABI for whole file.

Reply:
done